torch_np/_wrapper.py

```diff
@@ -179,14 +177,28 @@ def logspace(start, stop, num=50, endpoint=True, base=10.0, dtype=None, axis=0):
     return asarray(torch.logspace(start, stop, num, base=base, dtype=dtype))
 
 
-def arange(start, stop=None, step=1, dtype=None, *, like=None):
+def arange(start=None, stop=None, step=1, dtype=None, *, like=None):
```

Comment:
Why not setting `start=0`

Reply:
I did not manage to make all numpy tests pass otherwise. No doubt is can be fixed, but I'd rather move it to "not a first prio" bucket if you don't mind.